pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,10 @@ type Mux interface {
 type APIGroupVersion struct {
 	Storage map[string]rest.Storage
 
+	// Root is the APIPrefix under which this is being served.  It can also be the APIPrefix/APIGroup that is being served
```

Comment:
> Unfortunately this will be different from what I implemented in #14156. In #14156, Root just contains the APIPrefix. Group will be part of the Version, in the form of "group/version".

I prefer #14444 that keeps separate fields separate, rather than having a combined "version"